### ISSUE TITLE
Standard template block names

### ISSUE THREAD

**Issue by jpic** [codingjoe/django-cc#5]
In the Pinax ecosystem, apps template blocks are consistent across applications. Would this be something we want in django-cc ?

Here's the list of standard template block names:
- head_title: your base template can use this inside `<title>`
- extra_head: expected to be before `</head>`, ie. to add css
- body: that's the content of the page
- extra_body: expected to be before `</body>`, ie. to add js

Example: https://github.com/pinax/pinax-theme-pinaxproject/blob/master/pinax_theme_pinaxproject/templates/theme_base.html

**Comment by codingjoe** [comment]
I'm not so sure if this makes it easier, or more difficult. I understand that some projects that are heavy on templates include such a paragraph.

I'd like to keep this open, to see if we can find more supporters.

**Comment by jpic** [comment]
Oh, it's not necessarely for projects which are heavy on templates, but it does greatly increase the chances that they are compatible and predictible, which seemed somehow related to what we're trying to do here. Wait and see ;)

**Comment by codingjoe** [comment]
You know what, write a section. Let's see where this goes. Let's see at the end if the guide has become too massive. Maybe we have to release a short version than.

**Comment by jpic** [comment]
I agree that maybe we could work on a bit on the project's scope itself and then it'll be easy to know what must be in and what shouldn't, also agree that a cheatsheet would be nice.
